### ISSUE TITLE
Feature Request: Add browse button to model loader

### ISSUE THREAD

**Issue by ghost** [h2oai/h2ogpt#1166]
![image](https://github.com/h2oai/h2ogpt/assets/8894763/3f07b45e-2084-4b48-a17b-27ba6e46e25e)
It would be nice to add a simple browse button to select a model file directly.

**Comment by pseudotensor** [comment]
Yes, once gradio fixes critical gradio 4 bugs, I can use the file explorer they have.

https://www.gradio.app/docs/fileexplorer
